pkg/helper/client.go

```diff
@@ -251,6 +251,15 @@ func defaultGw() (string, error) {
 	})
 }
 
+// After running `nmstatectl set`, the last line contains the commit/rollback ID in format of:
+// Checkpoint: <ID>
+func readCommitID(setOutput string) string {
+	setOutputLines := strings.Split(strings.TrimSpace(setOutput), "\n")
+	commitIDLine := strings.TrimSpace(setOutputLines[len(setOutputLines)-1])
+	commitID := strings.Split(commitIDLine, " ")[1]
+	return commitID
```

Comment:
Can we do all this with a regex  and capturing group ? something like "^CheckPoint: (.*)$"

Reply:
I prefer basic tools if it is as simple as this. Regex is powerful but terrible to read.